internal/compiler/llr/optim_passes/inline_expressions.rs

```diff
@@ -99,6 +99,7 @@ fn builtin_function_cost(function: &BuiltinFunction) -> isize {
         BuiltinFunction::ShowPopupWindow | BuiltinFunction::ClosePopupWindow => isize::MAX,
         BuiltinFunction::SetSelectionOffsets => isize::MAX,
         BuiltinFunction::ItemMemberFunction(..) => isize::MAX,
+        BuiltinFunction::ItemFontMetrics => isize::MAX,
```

Comment:
is it really so bad to inline bindings that call this function? it should just be the cost of a few PROPERTY_ACCESS_COST, not? Or is it really necessary to always cache the result on its own property.

Reply:
Fair point, it's indeed not that bad. Just not instant (at least with Skia).